google_symptoms/delphi_google_symptoms/geo.py

```diff
@@ -92,5 +92,5 @@ def geo_map(df, geo_res, namescols =  None):
         newdf["geo_id"] = list(map_df.keys())[1:]
         mask = (newdf == 0)
         newdf[mask] = np.nan
-        converted_df = converted_df.append(newdf)
+        converted_df = pd.concat([converted_df, newdf])
```

Comment:
How many unique timestamps are typical? repeated calls to `concat` are substantially less performant than accumulating a bunch of dfs in a list and concatenating them together all at once. if it's doing this, like, 7 times it's probably fine, but if it's hundreds or thousands we should refactor to a list.

Reply:
We can have a lot of unique timestamps, e.g. on the occasional backfill run, so switching this to use a list then `concat` all at once.